packages/website/ts/pages/jobs/benefits.tsx

```diff
@@ -0,0 +1,83 @@
+import * as _ from 'lodash';
+import * as React from 'react';
+
+import { BulletedItemInfo, BulletedItemList } from 'ts/pages/jobs/bulleted_item_list';
+import { FilledImage } from 'ts/pages/jobs/filled_image';
+import { FloatingImage } from 'ts/pages/jobs/floating_image';
+import { colors } from 'ts/style/colors';
+import { ScreenWidths } from 'ts/types';
+
+const IMAGE_PATHS = ['/images/jobs/location1.png', '/images/jobs/location2.png', '/images/jobs/location3.png'];
+const BULLETED_ITEM_INFOS: BulletedItemInfo[] = [
+    {
+        bulletColor: '#6FCF97',
+        title: 'Ethics/Doing the right thing',
+        description: 'orem ipsum dolor sit amet, consectetur adipiscing elit.',
+    },
+    {
+        bulletColor: '#56CCF2',
+        title: 'Consistently ship',
+        description: 'orem ipsum dolor sit amet, consectetur adipiscing elit.',
+    },
+    {
+        bulletColor: '#EB5757',
+        title: 'Focus on long term impact',
+        description: 'orem ipsum dolor sit amet, consectetur adipiscing elit.',
+    },
+    {
+        bulletColor: '#6FCF97',
+        title: 'Test test yo',
+        description: 'orem ipsum dolor sit amet, consectetur adipiscing elit.',
+    },
+    {
+        bulletColor: '#56CCF2',
+        title: 'Waddle Waddle',
+        description: 'orem ipsum dolor sit amet, consectetur adipiscing elit.',
+    },
+];
+
+export interface BenefitsProps {
+    screenWidth: ScreenWidths;
+}
+
+export const Benefits = (props: BenefitsProps) => (
+    <div style={{ backgroundColor: colors.jobsPageBackground }}>
+        {props.screenWidth === ScreenWidths.Sm ? <SmallLayout /> : <LargeLayout />}
+    </div>
+);
+
+const LargeLayout = () => (
+    <div className="flex" style={{ height: 937 }}>
+        <div style={{ width: '43%', height: '100%' }}>
+            <ImageGrid />
+        </div>
+        <div style={{ paddingLeft: 205, width: '57%', height: '100%' }}>
+            <BenefitsList />
+        </div>
+    </div>
+);
```

Comment:
Do we need to use javascript to make the page responsive? I know it's how we're doing it elsewhere but just seems a bit overkill.

Also, I guess I'm not used to seeing all these fixed heights, and widths (even as percentages). Shouldn't it be enough to use a layout abstraction (responsive columns from basscss, or just flex flex-wrap) and let everything take up the space it needs (ie. less width / height rules, and more margin / padding rules).

Don't feel strongly about this if we need to ship this.

Reply:
Let's sync up in person about how you approach this layout